src/lib.rs

```diff
@@ -671,7 +684,7 @@ impl Section {
     ///
     /// `byte` may be a `Label` or a `u8`.
     /// Return this section.
-    pub fn D8<'a, T: ToLabelOrNum<'a, u8>>(mut self, byte: T) -> Section {
+    pub fn D8<'a, T: ToLabelOrNum<'a, u8>>(&mut self, byte: T) -> &mut Self {
```

Comment:
Any reason this (and the related methods) didn't get converted to use `byte: impl ToLabelOrNum<'a, u8>`?

Reply:
Ahh yes I can do that.